CLA.md

```diff
@@ -0,0 +1,66 @@
+### Scuffle Individual Contributor License Agreement
+
+Thank you for your interest in contributing to open source software projects (“Projects”) made available by Scuffle or its affiliates (“ScuffleTV”). This Individual Contributor License Agreement (“Agreement”) sets out the terms governing any source code, object code, bug fixes, configuration changes, tools, specifications, documentation, data, materials, feedback, information or other works of authorship that you submit or have submitted, in any form and in any manner, to Scuffle in respect of any of the Projects (collectively “Contributions”). If you have any questions respecting this Agreement, please contact opensource@scuffle.tv.
+
+
+You agree that the following terms apply to all of your past, present and future Contributions. Except for the licenses granted in this Agreement, you retain all of your right, title and interest in and to your Contributions.
+
+
+**Copyright License.** You hereby grant, and agree to grant, to Scuffle a non-exclusive, perpetual, irrevocable, worldwide, fully-paid, royalty-free, transferable copyright license to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute your Contributions and such derivative works, with the right to sublicense the foregoing rights through multiple tiers of sublicensees.
+
+
+**Patent License.** You hereby grant, and agree to grant, to Scuffle a non-exclusive, perpetual, irrevocable,
+worldwide, fully-paid, royalty-free, transferable patent license to make, have made, use, offer to sell, sell,
```

Comment:
Have made?

Reply:
This is straight from a legal template, i did not write this.